src/Persistence/functions.php

```diff
@@ -176,6 +201,24 @@ function enable_persisting(): void
     Configuration::instance()->persistence()->enablePersisting();
 }
 
+function assertPersisted(object $object, string $message = '{entity} is not persisted.'): void
```

Comment:
I feel like we should stick with our current snake_case function naming.

In a future PR, thinking we should add some helper phpunit methods to either Factories or a new trait:

- `->save()`
- `->assertPersisted()`
- etc...

Reply:
> I feel like we should stick with our current snake_case function naming.

do you mean `_assertPersisted()`?

Reply:
> do you mean I should change to _assertPersisted()?

`assert_persisted()` to match the other functions like `persistent_factory()`

Reply:
LOL ok sorry :sweat_smile: :facepalm: